### PR TITLE
Lesson 8

### PR DESCRIPTION
**Pull request by vtx123**

### CODE REVIEW

File: Lesson8/loop.hpp [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#ifndef loop_hpp
```
Comment (bohdanpc) [inline Lesson8/loop.hpp]: #pragma once

File: Lesson8/recursion.cpp [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+#include "recursion.hpp"
+
+void printNaturalNumbersGoingDown(int count){
+    if(count > 0){
+        std::cout << count << " ";
+        printNaturalNumbersGoingDown(--count);
```
Comment (bohdanpc) [inline Lesson8/recursion.cpp]: count - 1; No need to change the value at this point

Comment (vtx123) [reply Lesson8/recursion.cpp]: fixed

File: Lesson8/loop.hpp (hunk 1 above)
Comment (vtx123) [reply Lesson8/loop.hpp]: fixed

**Pull request merged**